x-pack/plugins/cases/docs/openapi/entrypoint.yaml

```diff
@@ -0,0 +1,90 @@
+openapi: 3.0.1
+info:
+  title: Cases
+  description: OpenAPI schema for Cases endpoints
+  version: '0.1'
+  contact:
+    name: Cases Team
+  license:
+    name: Elastic License 2.0
+    url: https://www.elastic.co/licensing/elastic-license
+tags:
```

Comment:
The tags here create the following view:

<img width="1278" alt="Screenshot 2022-05-05 at 8 43 44 AM" src="https://user-images.githubusercontent.com/7871006/166869032-418ab89e-a200-47e0-a70b-fecbb21fddd6.png">

It seems each tag creates a section. The endpoints are in the default section and the cases and kibana sections are empty. Maybe the endpoint should be under the Cases section without the kibana and default sections being presented?